ui/css/treeherder-navbar.css

```diff
@@ -426,6 +426,12 @@ fieldset[disabled] .btn-view-nav-closed.active {
   list-style: none;
 }
 
+.dropdown-menu li > button {
+  border: 0;
+  padding: 0;
+  background: transparent;
+}
+
```

Comment:
I think you can use bootstrap utilities for all of these properties: https://getbootstrap.com/docs/4.1/layout/utilities-for-layout/

Reply:
Ok, dropped these styles and used Bootstrap utilities instead.